public/index.js

```diff
@@ -90,5 +92,9 @@ Vue.createApp({
 			}
 			return climberData.certificate;
 		},
+		resetState() {
+			clearInterval(this.timer);
+			Object.assign(this.$data, this.$options.data.call(this));
```

Comment:
I guess `this.$options.data` refers to the first function in this file. Could you explain what's `this.$data`? Some internal state?

Reply:
Yes, this.$options.data is the first function and the $data is the internal state of the vue instance. Its a bit of a dense and hacky solution but then again is sure to catch all the edge cases in order to completely reset the state without reloading the page.